controller/metrics/clustercollector_test.go

```diff
@@ -65,8 +93,30 @@ argocd_cluster_connection_status{k8s_version="1.21",server="server1"} 0
 argocd_cluster_connection_status{k8s_version="1.21",server="server1"} 1
 argocd_cluster_connection_status{k8s_version="1.21",server="server2"} 1
 argocd_cluster_connection_status{k8s_version="1.21",server="server3"} 1
+
+# TYPE argocd_cluster_info gauge
+argocd_cluster_info{k8s_version="1.21",labels="env=dev",name="cluster1",server="server1"} 1
```

Comment:
For labels, I think it is better to add a new metric `argocd_cluster_labels`. You can replicate the logic that happens with `argocd_app_labels`, and the join can be done on the `server`. '

The application labels are controller with the `--metrics-application-labels` flag on the Argo CD application controller. You can add a `--metrics-cluster-labels` flags that will behave the same way.

Reply:
hi @agaudreault ,
okay, I did it